lte/gateway/c/core/oai/test/mock_tasks/mock_tasks.h

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+extern "C" {
+#include "dynamic_memory_check.h"
+#define CHECK_PROTOTYPE_ONLY
+#include "intertask_interface_init.h"
+#undef CHECK_PROTOTYPE_ONLY
+#include "intertask_interface.h"
+#include "intertask_interface_types.h"
+#include "itti_free_defined_msg.h"
+}
+
+const task_info_t tasks_info[] = {
+    {THREAD_NULL, "TASK_UNKNOWN", "ipc://IPC_TASK_UNKNOWN"},
+#define TASK_DEF(tHREADiD)                                                     \
+  {THREAD_##tHREADiD, #tHREADiD, "ipc://IPC_" #tHREADiD},
+#include <tasks_def.h>
+#undef TASK_DEF
+};
+
+/* Map message id to message information */
+const message_info_t messages_info[] = {
+#define MESSAGE_DEF(iD, sTRUCT, fIELDnAME) {iD, sizeof(sTRUCT), #iD},
+#include <messages_def.h>
+#undef MESSAGE_DEF
+};
+
+using ::testing::_;
```

Comment:
should we move these `using` declarations to the implementation file? Given that this will only be used on the test module I don't see much of a problem but just to avoid any source of namespace collusion, I'll leave it up to you.